### DISCUSSION TITLE
Mortgage counted double

### DISCUSSION THREAD

**Discussion by trumpimar** [firefly-iii/firefly-iii#6442] [Firefly III] [answered]
Hi, 

I have a question on how to properly implement mortgage payments as mentioned in https://github.com/firefly-iii/firefly-iii/issues/1956#issuecomment-480946458. The list mentions making a withdrawal from an asset account to the mortgage and then another withdrawal from the mortgage to the bank's expense account to split the payment into interest and repayment segments. This works for the balance of the mortgage, i.e. if I make a total payment of 1000 and 700 of that goes to interest I'll see the mortgage value increase by 300. 
However, when I look at any overview with expenses I'll see both the transactions being summed. So now my expense in the mortgage category look like it's 1700 instead of the 1000. What should I do to make sure the transaction doesn't get counted double?

**Comment by JC5** [comment]
Which overview exactly?

**Comment by trumpimar** [comment]
Either the category or transaction overviews or the category or default financial reports. I realize I could "decategorize" the transaction, but then I'll still see the 700 as an uncategorized expense making the sum of the "spent" columns incorrect.

**Comment by JC5** [comment]
I'm not seeing this. What version are you running?

**Comment by trumpimar** [comment]
5.7.10

**Comment by JC5** [comment]
When I make a split transaction like this one, it shows up fine everywhere.

<img width="1443" alt="Screenshot 2022-09-18 at 16 32 48" src="https://user-images.githubusercontent.com/5889984/190912373-803d01e5-7abf-4ccb-beff-ae8183ef1346.png">

What are you seeing?

**Comment by trumpimar** [comment, marked as answer]
OK, I get why that doesn't show up double. You're splitting a single transaction with part going to the bank and part going to the mortgage.
But in [the other issue](https://github.com/firefly-iii/firefly-iii/issues/1956#issuecomment-480946458) you show in the table that the full payment should go from asset account to the mortgage and then the interest payment from the mortgage to the bank's expense account, right? Or am I interpreting that table wrong?

**Reply by JC5** [reply]
That's about 2.5 years behind the times, sorry to say. Split the transaction, that should work just fine.

**Reply by trumpimar** [reply]
All right, will do. Just to be sure, there's still no way to automate making a split transaction?

**Reply by JC5** [reply]
Afraid not
